pkg/adapter/adapter.go

```diff
@@ -31,17 +36,141 @@ type envConfig struct {
 	MongoDbCredentialsPath string `envconfig:"MONGODB_CREDENTIALS" required:"true"`
 	Database               string `envconfig:"MONGODB_DATABASE" required:"true"`
 	Collection             string `envconfig:"MONGODB_COLLECTION" required:"false"`
+	EventSource            string `envconfig:"EVENT_SOURCE" required:"false"`
+	SinkURL                string `envconfig:"K_SINK" required:"false"`
```

Comment:
this should be required tue, for both of EventSource and SinkURL. BTW do you need SinkURL here? I thought adapter/v2 from eventing had that set?

Reply:
I think you are not using this SinkURL, is because it is being used in eventing. I think we should remove it from here if possible